packs/classfeatures/precise-strike.json

```diff
@@ -25,6 +25,41 @@
             "title": "Pathfinder Advanced Player's Guide"
         },
         "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.swashbuckler.preciseStrike.value",
```

Comment:
The only issue with this is people who already have this feature on their actors but have not taken bleeding finisher yet. We'll need to migrate this change since it would otherwise break on those actors.

Reply:
Yeah, I'm unsure how much of a hassle on the dev side that migration would be. It's certainly beyond me, so please feel free to close this if doing so is not a possibility.

Reply:
Shouldn't be a hard migration to write, maybe if I find some time this weekend I'll toss one up (unlikely that I will have time unfortunately)